### PR TITLE
Validate room name length

### PR DESCRIPTION
**Pull request by malaikaaiyar**
fixes https://github.com/sparkletown/internal-sparkle-issues/issues/476 
(only partly fixes because the existing over 20 character room names in the database need to be changed/handled)

How it looks:
Screen Shot 2021-03-25 at 9 40 47 PM

### CODE REVIEW

File: src/pages/Admin/Venue/DetailsValidationSchema.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export const validationSchema = Yup.object()
     name: Yup.string()
       .required("Required")
```
Comment (TroySK) [inline src/pages/Admin/Venue/DetailsValidationSchema.ts:60]: Unrelated to your change but would be great if these texts are replaced with more descriptive ones like in `src/pages/Admin/Details/ValidationSchema.ts`

**Review by TroySK — APPROVED** [review]
lgtm otherwise

**Review by mike-lvov — APPROVED** [review]
lgtm

File: src/pages/Admin/Venue/DetailsValidationSchema.ts (hunk 1 above)
Comment (malaikaaiyar) [reply src/pages/Admin/Venue/DetailsValidationSchema.ts:60]: I can make a new PR for this

Comment (0xdevalias) [reply src/pages/Admin/Venue/DetailsValidationSchema.ts:60]: See https://github.com/sparkletown/sparkle/pull/1297

**Review by 0xdevalias — COMMENTED** [review]
[Retroactive Review] Approved

**Pull request merged**